osbs/repo_utils.py

```diff
@@ -134,6 +134,16 @@ def __init__(self, dir_path='', depth=None, git_uri=None, git_branch=None, git_r
             possible_filename_typos=REPO_CONTENT_SETS_FILE_POSSIBLE_TYPOS
         )
 
+        if 'autorebuild' in self.container:
+            logger.user_warning("'autorebuild' config is deprecated in OSBS 2.0, this config is be "
```

Comment:
This should be (one of) "is being" or "will be"

"will be" is probably preferred in this context.

Same for the image_build_method warning below.

Reply:
I thought I had already fixed it. updated